chia-tools/Cargo.toml

```diff
@@ -26,3 +27,8 @@ bench = false
 name = "test-block-generators"
 test = false
 bench = false
+
+[[bin]]
+name = "compress-clvm"
```

Comment:
Not necessary here since it's done, but it might be easier in future to write command-line tools like this that are not sensitive to start-up time in python.